index.js

```diff
@@ -26,9 +33,9 @@ module.exports = function () {
 
     for (var i = 0; i < options.count; i++) {
       oldAllCallbacks = allCallbacks;
-      allCallbacks = [];
+      allCallbacks = {};
 
-      oldAllCallbacks.forEach(function (callback) {
+      entries(oldAllCallbacks).forEach(function ([ id, callback ]) {
```

Comment:
Could this be rewritten as:

```js
Object.keys(oldAllCallbacks).forEach(function (id) {
  var callback = oldAllCallbacks[id];
  callback(prevTime + options.time);
});
```

So that we don't need the extra lib + destructuring?